api/src/main/java/org/openmrs/module/fhir2/api/dao/impl/FhirPersonDaoImpl.java

```diff
@@ -75,7 +73,7 @@ protected void setupSearchParams(Criteria criteria, SearchParameterMap theParams
 	}
 	
 	@Override
-	protected Optional<Criterion> getCriteriaForLastUpdated(DateRangeParam param) {
+	protected Optional<Criterion> handleLastUpdated(DateRangeParam param) {
 		List<Optional<Criterion>> criterionList = new ArrayList<>();
 		
 		criterionList.add(handleDateRange("personDateVoided", param));
```

Comment:
Is it worth it adding a second param - `String property` to `handleLastUpdated()` method so that this functionality can be handled consistently by the superclass? Of course, you can override

Reply:
There are advantages to doing that, like I've actually left too much code in this `handleLastUpdated()` implementation. This seems like a one-off, though, as I'm not aware of other data-types that use non-standard fields for this.